### PR TITLE
Release 0.16.1

### PR DESCRIPTION
**Pull request by 2opremio**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,79 @@
...
+	[#1526](https://github.com/weaveworks/scope/issues/1526)
+- Force some known column widths to prevent truncation of others
+	[#1641](https://github.com/weaveworks/scope/pull/1641)
+
+Documentation:
+- Don't prepend `scope launch` with sudo
+	[#1606](https://github.com/weaveworks/scope/pull/1606)
+- Clarify instructions for using Scope with Weave Cloud
+	[#1611](https://github.com/weaveworks/scope/pull/1611)
+- readded signup page
```
Comment (rade) [inline CHANGELOG.md]: typo

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,79 @@
+## Release 0.16.1
+
+This is a bugfix release. In addition, the security of Scope probes can be
+hardened through the new flag `--probe.no-controls` with which controls will be
+disabled. Without controls, Scope users won't be able to open terminals,
+start/stop containers, show logs ...
```
Comment (rade) [inline CHANGELOG.md]: "In addition, the security of the Scope probe can be hardened by disabling controls with the new `--probe.no-controls` flag. Without controls, Scope users won't be able to open terminals, start/stop containers, view logs."

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,79 @@
...
+- weave cloud screen captures
+	[#1603](https://github.com/weaveworks/scope/pull/1603)
+
+Internal improvements and cleanup:
+- Promote fixprobe and delete rest of experimental
+	[#1646](https://github.com/weaveworks/scope/pull/1646)
+- refactor some timing helpers into a common lib
+	[#1642](https://github.com/weaveworks/scope/pull/1642)
+- Print DNS errors
+	[#1607](https://github.com/weaveworks/scope/pull/1607)
```
Comment (rade) [inline CHANGELOG.md]: this looks like a user-visible/relevant change.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,79 @@
...
+- Make it easier to disable weave integrations
+	[#1610](https://github.com/weaveworks/scope/pull/1610)
```
Comment (rade) [inline CHANGELOG.md]: this looks like a user-visible/relevant change.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,79 @@
...
+- Add dry run flag to scope, so when launched we can check the args are valid.
+	[#1609](https://github.com/weaveworks/scope/pull/1609)
```
Comment (rade) [inline CHANGELOG.md]: this looks like a user-visible/relevant change.

**Comment by 2opremio** [comment]
I've tested it:
* Locally
* Connecting a probe to the service (backwards compatibility)
* Monitoring the service locally
* In ECS

**Comment by 2opremio** [comment]
I need LGTMs from some (ideally all) of @errordeveloper @jml @tomwilkie @paulbellamy a @foot

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,90 @@
+# Release 0.16.1
+
+This is a bugfix release. In addition, the security of the Scope probe can be
+hardened by disabling controls with the new `--probe.no-controls` flag. Without
+controls, Scope users won't be able to open terminals, start/stop containers,
+view logs, etc.
```
Comment (rade) [inline CHANGELOG.md]: tightening this slightly... "In addition, the security of the Scope probe can be hardened by disabling controls with the new `--probe.no-controls` flag, which prevents users from opening terminals, starting/stopping containers, viewing logs, etc."

Comment (2opremio) [inline CHANGELOG.md]: Thanks, reads  better like that.

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,3 +1,89 @@
+# Release 0.16.1
+
+In addition, the security of the Scope probe can be hardened by disabling
```
Comment (jml) [inline CHANGELOG.md]: "In addition" is an unusual way to start. Perhaps remove it?

**Comment by jml** [comment]
Only issue I found was #1678 and that's not a release blocker.

I've made a suggestion re working on the changelog, but I don't think that's a blocker either.

LGTM.

**Comment by paulbellamy** [comment]
LGTM to me as well.

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,3 +1,91 @@
...
+	[#1669](https://github.com/weaveworks/scope/pull/1669)	
+- Fixes terminal wrapping by syncing docker/term.js terminal widths.
+	[#1648](https://github.com/weaveworks/scope/pull/1648)
+- Wrongly attributed local side in outbound internet connections
+	[#1598](https://github.com/weaveworks/scope/issues/1598)
+- Cannot port-forward app from kubernetes with command in documentation
+	[#1526](https://github.com/weaveworks/scope/issues/1526)
+- Force some known column widths to prevent truncation of others
+	[#1641](https://github.com/weaveworks/scope/pull/1641)
+- Check if app and probe are running first
```
Comment (2opremio) [inline CHANGELOG.md]: the changelog should contain the text in the issue, will fix

File: CHANGELOG.md (hunk 7 above)
Comment (2opremio) [inline CHANGELOG.md]: It used to be: _Bugfix release. In addition ..._ I removed the first phrase by error when addressing a PR comment. Will fix

**Comment by foot** [comment]
LGTM too

**Comment by errordeveloper** [comment]
LGTM

**Pull request merged**